6-reading_number.py

```diff
@@ -0,0 +1,6 @@
+def reading_number():
+    dict_number={1:'one', 2:'two' , 3:'three' , 4:'four', 5:'five', 6:'six', 7:'seven', 8:'eight', 9:'nine', 10:'ten', 11:'eleven' , 12:'twelf'}
+    a=input(int('please enter a number'))
+    if a in dict_number.keys():
+        print(dict_number.values[a])
+reading_number()
```

Comment:
- kodlarınız hata veriyor. 5. satırda dict_number.values[a] ifadesini dict_number[a] olarak değiştirirseniz problem çözülür ama yaptığınız çözüm istenen çözümden biraz uzak. soruda inpu olarak alınan 2 haneli rakamların yazı ile yazdırılması istenmiş. 10 ile 99 arası. sizinki sadece 12 ye kadar olanı verebiliyor.
örnek bir çözüm aşağıdadır.

def reading_number(two_digit_number=''):
    """gives the reading of entered two-digit number."""
    exceptional_numbers = ('Ten', 'Eleven', 'Twelve', 'Thirteen', 'Fourteen', 'Fifteen',
                           'Sixteen', 'Seventeen', 'Eighteen', 'Nineteen')
    units = ('', 'One', 'Two', 'Three', 'Four', 'Five', 'Six', 'Seven', 'Eight', 'Nine')
    tens = ('', 'Ten', 'Twenty', 'Thirty', 'Forty', 'Fifty', 'Sixty', 'Seventy', 'Eighty', 'Ninety')
    if str(two_digit_number).isnumeric() is False:
        return 'All characters of the input must be number!'
    elif int(two_digit_number) < 10 or int(two_digit_number) > 99:
        return 'This function returns only two digit numbers'
    elif int(two_digit_number) < 20:
        return exceptional_numbers[int(str(two_digit_number)[1])]
    return tens[int(str(two_digit_number)[0])] + ' ' + units[int(str(two_digit_number)[1])]


print(reading_number(input('Please enter a two-digit number: ')))